### DISCUSSION TITLE
What's the status of vite?

### DISCUSSION THREAD

**Discussion by BryceBlankinship** [remix-run/remix#8356] [General]
Hey remix team, what is the status of vite becoming stable? I had a bunch of issues trying to migrate to it for a somewhat large project of mine. Would like to get that HMR as my laptop only has 8gb of ram so build times are around 12 seconds... Which is painfully slow. (I have a macbook pro M1, shouldn't be this slow but I'm hoping more ram will speed things up). Is there anything I can do to help the framework move onto vite? I'm open to learning and contributing.

**Comment by kiliman** [comment]
The Vite plugin is "_unstable_" only in the sense that the API surface may change, so you may end up changing your code slightly on each release. There have been some breaking changes recently, but it was mainly how to integrate with custom Node servers. But otherwise, the Vite plugin works very well, and I haven't had any issues.

I would definitely try migrating your project to Vite.

I have a Remix+Vite template that adds Tailwind, Vitest, Playwright, etc. The docs folder has instructions on how each part is configured so you can pick and choose what you want to include if you don't want the full template.

https://github.com/kiliman/remix-vite-template
